src/slic3r/GUI/Preferences.cpp

```diff
@@ -1021,6 +1021,7 @@ wxWindow* PreferencesDialog::create_general_page()
 
     auto item_mouse_zoom_settings = create_item_checkbox(_L("Zoom to mouse position"), page, _L("Zoom in towards the mouse pointer's position in the 3D view, rather than the 2D window center."), 50, "zoom_to_mouse");
     auto item_use_free_camera_settings = create_item_checkbox(_L("Use free camera"), page, _L("If enabled, use free camera. If not enabled, use constrained camera."), 50, "use_free_camera");
+    auto reverse_mouse_zoom = create_item_checkbox(_L("Reverse mouse zoom"), page, _L("If enabled, reverses the direction of zoom with mouse wheel."), 50, "reverse_mouse_wheel_zoom");
```

Comment:
We should set reverse_mouse_wheel_zoom to false by default in AppConfig

Reply:
Got it.

Reply:
I've just looked at the code and remember being wonder that it is already there: https://github.com/SoftFever/OrcaSlicer/blob/b38133fd90a47e630a82a657312171e2263e683e/src/libslic3r/AppConfig.cpp#L177

Reply:
> I've just looked at the code and remember being wonder that it is already there:
> 
> https://github.com/SoftFever/OrcaSlicer/blob/b38133fd90a47e630a82a657312171e2263e683e/src/libslic3r/AppConfig.cpp#L177

👍